static/js/containers/pages/DashboardPage.js

```diff
@@ -194,53 +184,60 @@ export class DashboardPage extends React.Component<
   renderEmailSettingsDialog(enrollment: RunEnrollment) {
     const { emailSettingsModalVisibility } = this.state
     return (
-      <Modal
-        className="text-center"
-        isOpen={emailSettingsModalVisibility}
-        toggle={this.toggleEmailSettingsModalVisibility}
+      <div
+        className="modal fade"
+        id={`enrollment-${enrollment.id}-modal`}
+        role="dialog"
       >
-        <ModalHeader toggle={this.toggleEmailSettingsModalVisibility}>
-          Email Settings for {enrollment.run.course_number}
-        </ModalHeader>
-        <ModalBody>
-          <Formik
-            onSubmit={this.onSubmit.bind(this)}
-            initialValues={{
-              subscribeEmails: enrollment.edx_emails_subscription,
-              enrollmentId:    enrollment.id,
-              courseNumber:    enrollment.run.course_number
-            }}
-            render={({ values }) => (
-              <Form className="text-center">
-                <section>
-                  <Field
-                    type="hidden"
-                    name="enrollmentId"
-                    value={values.enrollmentId}
-                  />
-                  <Field
-                    type="hidden"
-                    name="courseNumber"
-                    value={values.courseNumber}
-                  />
-                  <Field
-                    type="checkbox"
-                    name="subscribeEmails"
-                    checked={values.subscribeEmails}
-                  />{" "}
-                  <label check>Receive course emails</label>
-                </section>
-                <Button type="submit" color="success">
-                  Save Settings
-                </Button>{" "}
-                <Button onClick={this.toggleEmailSettingsModalVisibility}>
-                  Cancel
-                </Button>
-              </Form>
-            )}
-          />
-        </ModalBody>
-      </Modal>
+        <div className="modal-dialog">
```

Comment:
This is however showing the correct course number on the title but it has introduced some issues alongside.

1) The issue is that the model is not maintaining the state properly now.

**Reproduction steps:**
Click on `Email Settings` in the menu for any enrolled course, Uncheck the `Receive Course emails` box and click the close button instead of clicking save. Now when reopened this should still be because the user didn't save the settings but instead it staus unchecked.

2) I don't understand why we removed the <Modal> completely and went to our own html design?